xline-client/src/types/auth.rs

```diff
@@ -47,15 +52,15 @@ impl AuthUserAddRequest {
         }
     }
 
-    /// Set password.
+    /// Sets the password.
     #[inline]
     #[must_use]
     pub fn with_pwd(mut self, password: impl Into<String>) -> Self {
         self.inner.password = password.into();
         self
     }
 
-    /// Set no password.
+    /// If set, Xline will add the user without password, password set with `with_pwd` will be ignored.
     #[inline]
     #[must_use]
     pub const fn with_no_pwd(mut self) -> Self {
```

Comment:
Is it mandatory for a user to set his password? If not, why don't we default to no password unless the user explicitly invokes `with_pwd`?

Reply:
makes sense, I removed the with_no_pwd method and set it as default.